src/components/Articles.jsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react"
+import { ApiArticles } from "../Api"
+
+function Articles() {
+
+    const [articles, setArticles] = useState();
+
+    useEffect(() => {
+        ApiArticles().then((pendingArticles) => {
+            setArticles(pendingArticles.articles)
+        })
+    }, []);
+    if (!articles) {
+        return <p>loading...</p>
```

Comment:
Ideally this should be using a loading state rather than relying on the article data to be there or not.